### DISCUSSION TITLE
Log received messages to a file for later examination in high volume cases

### DISCUSSION THREAD

**Discussion by Royscw** [mockoon/mockoon#700] [Ideas] [answered]
I have a very strong need to have the received messages logged to a file so that they can be shared and examined at a later date. Looking at the logs within the application is very clunky when we are having to check all of the messages.

Could this please be looked at? It seems like a simple change to output the received messages to a log file for each of the endpoints. This would save a huge amount of work (its used a lot) and would mean we can keep using the tool. Without this functionality we must unfortunately go elsewhere to ease the burden on our testing.

I hope this makes sense! 
Thanks,
Steve

**Comment by 255kb** [comment, marked as answer]
It's true that both desktop and CLI are using the same internal lib, so, logging in both the UI and log files in the desktop app should be possible without too much work. 

Could you please open an issue to track this feature request? Thank you!
